src/content/docs/en/tutorials/add-content-collections.mdx

```diff
@@ -248,6 +248,7 @@ The steps below show you how to extend the final product of the Build a Blog tut
     const allPosts = await Astro.glob("../pages/posts/*.md");
     const allPosts = await getCollection("posts");
     ---
+    ```
```

Comment:
I am confused to see how quote block functions just well without closing itself

Reply:
I am very confused too!  :sweat_smile: